drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-backend/src/main/java/org/drools/workbench/screens/guided/dtable/backend/server/conversion/XLSBuilder.java

```diff
@@ -53,9 +53,12 @@ public BuildResult build() {
             makeImports();
 
             makeTable();
-        } catch (final Exception e) {
+        } catch (final UnsupportedOperationException e) {
             return new BuildResult(workbook,
                                    new XLSConversionResult(e.getMessage()));
+        } catch (final Exception e) {
```

Comment:
Why we added this catch block please? I do not see new `throw new` statement? Just discussing, are you aware of the place that cause this catch block is needed or is it just to be hundred percent sure?

Reply:
It is just to be sure. Even if this feature was complete. We still have old GDST formats that are likely to cause trouble. So this I'm hoping this picks them up and gives a cleaner report.

Reply:
ok